src/tests/app.test.js

```diff
@@ -267,6 +267,39 @@ describe('Broadcaster tests:', () => {
         done();
       });
   });
+  it('update incident status', (done) => {
+    chai
+      .request(app)
+      .patch(`/api/v2/incident/${id}/status`)
+      .set('token', dummyData.adminToken)
+      .send(dummyData.incident)
+      .end((err, res) => {
+        chai.expect(res.status).to.eq(200);
+        done();
```

Comment:
Hey @shaazk can you expect on some of the values being returned as part of `res.body`? Ensure that you not only test the status code, but also the component of the response you return to use.

Reply:
Noted, let me get to it